scripts/auth.yml.example

```diff
@@ -1,2 +1,5 @@
 google:
     key: YOUR_KEY
+pypi:
```

Comment:
This makes me nervous (accidental commit of credentials to github) - does ~/.pypirc not work with twine?